src/api/app/controllers/webui/package_controller.rb

```diff
@@ -81,6 +81,7 @@ def show
 
     @comments = @package.comments
     @requests = []
+    @services = Service.find(project: @project.name, package: @package.name)
```

Comment:
IMHO we should detect _service files while indexing the source data in backend_package data.So it would be a simple (and cached) database lookup and no need for transfering data from source server.

Check for example is_patchinfo? method from package model.